addons/website_sale_picking/tests/test_ui.py

```diff
@@ -51,3 +52,99 @@ def test_onsite_payment_tour(self):
         self.env.ref("website_sale_picking.payment_provider_onsite").is_published = True
 
         self.start_tour('/shop', 'onsite_payment_tour')
+
+    def test_onsite_payment_fiscal_change_tour(self):
+        # Make sure at least one onsite payment option exists.
+        if not loaded_demo_data(self.env):
+            _logger.warning(
+                "This test relies on demo data. To be rewritten independently of demo data for accurate and reliable results.")
+            return
```

Comment:
Mhm, we're not supposed to add new tests relying on demo data.  The `loaded_demo_data` tooling objective is to be progressively removed everywhere when tests are improved.